lib/tasks/hydranorth-solr.rake

```diff
@@ -6,6 +6,29 @@ require 'rdf/turtle'
 namespace :hydranorth do
   namespace :solr do
 
+    desc "Index objects by batch files"
```

Comment:
I will add a sample file and updated the README - this is to do reindex from solr's output.